packages/api/src/command/medical/cq-directory/rebuild-cq-directory.ts

```diff
@@ -0,0 +1,65 @@
+import { Carequality } from "@metriport/carequality-sdk/client/carequality";
+import { sleep } from "@metriport/core/util/sleep";
+import { createMockCQOrganization } from "../../../external/carequality/organization-mock";
+import { Config } from "../../../shared/config";
+import {
+  CQDirectoryEntryDataWithUpdateAndId,
+  createOrUpdateCQDirectoryEntries,
+} from "./create-cq-directory-entry";
+import { parseCQDirectoryEntries } from "./parse-cq-directory-entry";
+import { capture } from "../../../shared/notifications";
+
+const BATCH_SIZE = 300;
+
+type CQDirectoryRebuildResponse = {
+  totalFetched: number;
+  added: number;
+  updated: number;
+};
+
+export const rebuildCQDirectory = async (
```

Comment:
maybe break this up into smaller functions? 
a `createMockOrganizations` function for the mock logic might be enough, though I know you said mock stuff was temporary so maybe you can ignore this. Your call